crates/crypto/bls/src/zkcrypto/pubkey.rs

```diff
@@ -27,3 +31,19 @@ impl TryFrom<&PubKey> for G1Affine {
         }
     }
 }
+
+impl Aggregatable<PubKey> for PubKey {
+    type Error = BLSError;
+
+    fn aggregate(pubkeys: &[&PubKey]) -> Result<PubKey, Self::Error> {
+        let aggregate_point = pubkeys
+            .iter()
+            .try_fold(G1Projective::identity(), |acc, pubkey| {
+                Ok(acc.add(&G1Projective::from(G1Affine::try_from(*pubkey)?)))
```

Comment:
```suggestion

    fn aggregate(public_keys: &[&PubKey]) -> Result<PubKey, Self::Error> {
        let aggregate_point = public_keys
            .iter()
            .try_fold(G1Projective::identity(), |acc, public_key| {
                Ok(acc.add(&G1Projective::from(G1Affine::try_from(*public_key)?)))
```

what is acc can you rename that to something more descriptive